File: 02. C# Part2/03. Methods-Homework/05. LargerThanNeighbours/LargerThanNeighbours.cs
using System;
using System.Linq;

        //Write a method that checks if the element at given position in given array of integers 
        //is larger than its two neighbours (when such exist).

     public class LargerThanNeighbours
    {
        static void Main()
        {
            Console.WriteLine("Enter the integers of the array: ");
            int[] numbers = Console.ReadLine()
                .Split(new char[] { ' ', ',', '\t' }, StringSplitOptions.RemoveEmptyEntries)
                .Select(x => int.Parse(x))
                .ToArray();
            Console.WriteLine("Enter the position of the number in the array: ");
            int position = int.Parse(Console.ReadLine());
            if (position > numbers.Length)
            {
                Console.WriteLine("There is no such position.");
            }
            bool isLarger = IsTheNumberLarger(numbers, position);
            Console.WriteLine("Is the number larger? {0}", isLarger);  
        }

        public static bool IsTheNumberLarger(int[] numbers, int position)
        {
            for (int i = 0; i < numbers.Length; i++)
            {
                if (numbers[position] == numbers[numbers.Length - 1] || numbers[position] == numbers[0])
                {
                    Console.WriteLine("The number doesn't have two neighbours.");
                    break;
                }
                if (numbers[position] > numbers[position + 1] && numbers[position] > numbers[position - 1])
                {
                    return true;
                }
            }
            return false;
        }
    }

File: 06. JavaScript Basics/06. Functions-Homework/06. Functions-Homework/06. LargerThanNeighbours/LargerThanNeighbours.js
//Write a function that checks if the element at given position in given array of integers is bigger than its two neighbours 
//(when such exist).

function largerThanNeighbours(arr, position) {
    for (var i = 0; i < arr.length; i++) {
        if (position <= 0 || position >= arr.length - 1) {
            jsConsole.writeLine("Please take another element's position.");
            break;
        }
        else {
            if (arr[position] > arr[position - 1] && arr[position] > arr[position + 1]) {
                jsConsole.writeLine("True");
                break;
            }
            else {
                jsConsole.writeLine("False");
                break;
            }
        }
    }
}

largerThanNeighbours([1, 2, 8, 6, 89, 2, 5, 3, 1], 6);